<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>648</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>648</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>648</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="LoadFuncView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="FileListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>91</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
   </widget>
   <widget class="QListView" name="AppearencesListView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="AppearencePropFuncListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>731</width>
      <height>271</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ChooseFolderBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="FilterLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FilterComboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>_G.STATE_IDLE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>_G.STATE_WALK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>_G.STATE_DEAD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>_G.STATE_FIRE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>_G.STATE_CUSTOM</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="FilterLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FilterFuncComboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearFuncFilterButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAppFilterButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
